pkgs/development/libraries/AntTweakBar/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl, unzip, xlibs, mesa }:
+
+stdenv.mkDerivation rec {
+  name = "AntTweakBar-1.16";
+
+  buildInputs = [ unzip xlibs.libX11 mesa ];
+
+  src = fetchurl {
+    url = "mirror://sourceforge/project/anttweakbar/AntTweakBar_116.zip";
+    sha256 = "0z3frxpzf54cjs07m6kg09p7nljhr7140f4pznwi7srwq4cvgkpv";
+  };
+
+  postPatch = "cd src";
+  installPhase = ''
+    mkdir -p $out/lib/
+    cp ../lib/{libAntTweakBar.so,libAntTweakBar.so.1,libAntTweakBar.a} $out/lib/
+    cp -r ../include $out/
+  '';
+
+  meta = {
+    description = "Add a light/intuitive GUI to OpenGL applications";
+    longDescription = ''
+      A small and easy-to-use C/C++ library that allows to quickly add a light
+      and intuitive graphical user interface into graphic applications based on OpenGL
+      (compatibility and core profiles), DirectX 9, DirectX 10 or DirectX 11
+      to interactively tweak parameters on-screen
+    '';
+    homepage = http://anttweakbar.sourceforge.net/;
+    license = stdenv.lib.licenses.zlib;
+    maintainers = stdenv.lib.maintainers.razvan;
```

Comment:
Oh right, borg is right: this should be a list of strings.

Reply:
@7c6f434c Darn, and I thought I was being smart about it by making it non-list :( Makes sense though to be a list

Reply:
@7c6f434c Ironically platforms does not have to be a list haha

Reply:
They do; but Linux is not a single platform, it is a list of at least three different ones for practical purposes (`i686`, `x86_64`, `aarch64`), and then there are a few 32-bit ARM ones and a 64-bit MIPS one.

Reply:
@7c6f434c So it is ok the way it is now, right?